### ISSUE TITLE
log file

### ISSUE THREAD

**Issue by idnovic** [SamSchott/maestral-dropbox#10]
Please add a log file.
You do mention to test all use cases with maestral but I would not know that something went wrong without a logfile to look at. Just log errors.

**Comment by idnovic** [comment]
notes for myself 

https://github.com/Delgan/loguru

**Comment by idnovic** [comment]
Am I under the right impression that maestral does load a logger function but does not log output to a file?
I took a look at the source code but I am kinda new to python programming.

**Comment by SamSchott** [comment]
Yes, it already uses logging but messages are just printed to stdout at the moment. Up to v0.1.2, "debug" was the default logging level, but I switched to "info" now that sync errors are less common. There is currently no command line flag to select the logging level, but it can be changed easily when running Maestral in an interactive Python session.

I will consider creating a log file, once proper error handling has been implemented. But my preferred solution would be to handle all expected errors (that are not bugs) through the logger with level "warning" or higher notify the user directly as appropriate. See #5.
